src/frontend/src/catalog/system_catalog/pg_catalog/mod.rs

```diff
@@ -297,7 +300,14 @@ impl SysCatalogReaderImpl {
                     Row::new(vec![
                         Some(ScalarImpl::Int32(index.id.index_id() as i32)),
                         Some(ScalarImpl::Int32(index.primary_table.id.table_id() as i32)),
-                        Some(ScalarImpl::Int16(index.index_item.len() as i16)),
+                        Some(ScalarImpl::Int16(index.original_columns.len() as i16)),
+                        Some(ScalarImpl::List(ListValue::new(
+                            index
+                                .original_columns
+                                .iter()
+                                .map(|index| Some(ScalarImpl::Int16(index.get_id() as i16 + 1)))
```

Comment:
Indexing starts at 1? Is this the behavior in PostgreSQL?

Reply:
Yes

Reply:
> indkey int2vector (references [pg_attribute](https://www.postgresql.org/docs/current/catalog-pg-attribute.html).attnum)
> This is an array of indnatts values that indicate which table columns this index indexes. For example, a value of 1 3 would mean that the first and the third table columns make up the index entries. Key columns come before non-key (included) columns. A zero in this array indicates that the corresponding index attribute is an expression over the table columns, rather than a simple column reference.